2.0/UnlitTest/README.md

```diff
@@ -0,0 +1,21 @@
+# Unlit Test
+
+## Screenshot
+
+![screenshot](screenshot/screenshot_large.jpg)
+
+## Description
+
+This model tests the [`KHR_materials_unlit`](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit) extension.  There are two objects, one orange-ish (`#FF7F00` in sRGB space) and one blue-ish (`#007FFF` in sRGB space).  The visible surface of these two objects should appear as a uniform color, so much so that the 3D effect is intentionally lost.
+
+## Problem: Lighting applied to un-lit model
+
+![screenshot](screenshot/unlit_test_fail.jpg)
```

Comment:
This image includes shadows. It might get confused with the "lighting effects are visible" statement. If you are planning on updating the screenshot, it might be better to do it without the shadows. If you want, you can easily turn off the unlit extension in the BabylonJS sandbox via the inspector.

![image](https://user-images.githubusercontent.com/19158740/50849510-1a2a9d80-132c-11e9-9452-101ef3bf3fb9.png)

Reply:
Cool.  Off-topic, can this change be applied in VSCode?  At the bottom it says I need to reload the model, but in VSCode that reloads the entire webview, including resetting the inspector.

Reply:
Also, on the Babylon Sandbox website, when I turn this off, then it (correctly) gives me the "extension not supported" error message 😄

Reply:
> can this change be applied in VSCode?

I was just thinking about that. We need a button to reload the asset for this to work.

Reply:
> Also, on the Babylon Sandbox website, when I turn this off, then it (correctly) gives me the "extension not supported" error message

Yeah, you will have to temporarily remove the `extensionsRequired` part.

Reply:
Actually I think we should keep the shadows.  I got rid of the red text above, and the README below already talks about the shadow being allowed.

Reply:
Oh, I missed that. Sounds fine then.